### PR TITLE
add option to run only unit test

### PR DESCRIPTION
**Pull request by michaelamattes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ jobs:
         uses: ./terratest-action
         with:
           test:  azure
+          test_stack: unit
```
Comment (rndmh3ro) [inline README.md:54]:
```suggestion
          test_type: unit
```

Comment (michaelamattes) [reply README.md:54]: yes sounds better,  i'll change

**Review by rndmh3ro — APPROVED** [review]

File: action.yaml [hunk 2]
```diff
@@ -5,6 +5,9 @@ inputs:
   test:
     description: 'name of the test to run (azure)'
     required: true
+  TEST_TYPE:
```
Comment (schurzi) [inline action.yaml:8]: why is this in caps?

Comment (michaelamattes) [reply action.yaml:8]: copy&paste error fixed with https://github.com/T-Systems-MMS/terratest-action/pull/10

**Pull request merged**